### DISCUSSION TITLE
Domains for Tenants(Teams)

### DISCUSSION THREAD

**Discussion by isrardogar** [filamentphp/filament#10192] [Help]
Can we add Domains for Tenants ?
There is documentation to add domain to Panel.
can we add different domains for each tenant ?

**Comment by vitalijalbu** [comment]
would be great... also a different database too...

**Reply by isrardogar** [reply]
As if we build saas app we need tenants, and tenants need their own domains.
i have been using stancl tenancy for some time it's a great package for Multi Tenancy.

**Reply by citricguy** [reply]
Have you been able to integrate with Filament v3?

**Reply by isrardogar** [reply]
not yet
